sui/deploy-contract.js

```diff
@@ -269,16 +271,17 @@ async function upgrade(keypair, client, supportedPackage, policy, config, chain,
     const contractConfig = contractsConfig?.[packageName];
 
     validateParameters({ isNonEmptyString: { packageName } });
+    const moveDir = `${__dirname}/move`;
```

Comment:
but this dir doesn't contain a copy of move contracts, how would it work?

Reply:
it's generated after the package deployment [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/57839b280aff4670996f2ac95218b5970f1d5e1e/sui/utils/utils.js#L63-L65)

Reply:
I see